### ISSUE TITLE
Code review request

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BenHenning/oppia#204]
```
Branch name: documentation


Links to commits:

(1) 
https://code.google.com/p/oppia/source/detail?r=336755cfe0fc3b18034d8ee78f205295
24929fd3&name=documentation

(2) 
https://code.google.com/p/oppia/source/detail?r=4e73ec57d573e4c63cec24fb39b33465
4d7c128a&name=documentation


Purpose of code changes on this branch:

Add documentation about the beta/release criteria; rewrite the publish modal; 
rename 'playtest gallery' to 'playtest queue'. Please note that changes to the 
publish modal in (2) overwrite changes to the same modal in (1).

Not done:
- full audit of all terminology on existing About pages
- text changes on exploration editor page (I wasn't sure what exactly these 
should be)
- gallery descriptions (not sure if there's more to do than just the renaming?)
- notification/butterbar on editor page
- About pages on the overview, common use cases and playtesting


When reviewing my code changes, please focus on:

Wording (both style and accuracy).


General notes:
- This branch has already been merged into 'develop' and a cut is going to be 
made tonight for the v1.2.4 release. Please review as soon as you can!
- I am assuming that this is sufficient to release in v1.2.4 as-is. If there is 
anything further that has to be done before we actually upgrade the site, 
please let me know as soon as possible. If you like, it may also be easier to 
just create a branch off of release/1.2.4 and make edits on that, then send it 
to me for review + merge.

Thanks!

```

Original issue reported on code.google.com by `s...@google.com` on 3 May 2014 at 1:09
